src/main/java/com/robin/hoteless/Reservation.java

```diff
@@ -0,0 +1,42 @@
+package com.robin.hoteless;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+public class Reservation {
+    int id;
+    Date made;
+    int stayDays;
+    double dailyRate;
+    LocalDate in;
+    LocalDate out;
+    String bedType;
+    double total;
+    static int totalReservations;
```

Comment:
This field would not belong here, as it is an aggregate of reservations...which is this class. Or perhaps the naming is the problem, and it should be something more like ``guestCount``?